### PR TITLE
Behavior.Keyboard - hot keys!!!

### PR DESCRIPTION
**Pull request by anutron**
* Adds new feature support for MooTools’ Keyboard
* Also invokes the destroyDom event for the `nix` trigger

### CODE REVIEW

File: dist/dev/js/behavior-ui-standalone.js [hunk 1]
```diff
@@ -10458,6 +10458,7 @@ name: Delegator.Fx.Reveal
         }
```
Comment (anutron) [inline dist/dev/js/behavior-ui-standalone.js]: build artifact

File: dist/dev/js/behavior-ui.js [hunk 2]
```diff
@@ -22912,6 +22912,7 @@ name: Delegator.Fx.Reveal
         }
```
Comment (anutron) [inline dist/dev/js/behavior-ui.js]: build artifact

**Comment by anutron** [comment]
@davywentworth I can't attach a screenshot here because all the action happens with keyboard interaction. Nutshell: you can now assign hot keys to any delegator trigger. You can either point it at an element with a delegator trigger already on it and invoke it (i.e. there's a link w/ trigger=ajax and you can assign a hot key to invoke it as if it were clicked) OR you can define the entire thing in the keyboard configuration (i.e. there's a plain link, and you could assign a hot key to load it w/ ajax).

**Comment by davywentworth** [comment]
🐑

**Comment by anutron** [comment]
@davywentworth two things:

1. The way to specify keys is defined in Keyboard's docs in MooTools. There's a separate task here to make a simple behavior that will show the user all the keyboard commands currently active. I'll put some thought into how to express that.
2. I made a change since this pull which I won't ask you to review that basically adds the ability for you to just do this:

```
<a class="btn btn-default" data-trigger="toggleClass"
       data-toggleclass-options="'class':'btn-primary'"
       data-keyboard-key="shift+a">hit shift+a to toggle class</a>
```

I.e. you can just specify a key for an element and it'll simulate a click, invoking all its delegators.

**Pull request closed without merging**